src/querybuilder.ts

```diff
@@ -134,6 +135,23 @@ export class QueryBuilder {
     return this;
   }
 
+  public replace(data: QueryValues): QueryBuilder {
```

Comment:
Please add a TSDoc comment to this method.

Example:

```
/**
 * Perform `REPLACE` query to the table.
 * 
 * It will look for `PRIMARY` and `UNIQUE` constraints.
 * If something matched, it gets removed from the table
 * and creates a new row with the given values.
 */
public replace(): QueryBuilder {
```